packages/mongo-nav/src/types.ts

```diff
@@ -1,18 +1,50 @@
+const ActiveNavElement = {
```

Comment:
We should update all usages of `activeNav === XX` to use this object instead, as that's the object that dictates what in the nav can be set as active.

Reply:
👍